stuffer/s2n_stuffer.c

```diff
@@ -162,17 +162,12 @@ void *s2n_stuffer_raw_read(struct s2n_stuffer *stuffer, uint32_t data_len)
 
 int s2n_stuffer_read(struct s2n_stuffer *stuffer, struct s2n_blob *out)
 {
-
-    GUARD(s2n_stuffer_skip_read(stuffer, out->size));
-
-    void *ptr = stuffer->blob.data + stuffer->read_cursor - out->size;
-    if (ptr == NULL) {
+    // null check the out structure
+    if (NULL == out) {
```

Comment:
please use notnull_check(out), which is the pattern used thoughout s2n for error checking.